triagebot.toml

```diff
@@ -113,7 +113,7 @@ topic = "P-critical #{number} {title}"
 message_on_add = "@*WG-prioritization* issue #{number} has been assigned `P-critical`."
 
 [notify-zulip."P-high"]
-required_labels = ["regression-from-stable-to-*"]
+required_labels = ["regression-from-stable-to-[bn]*"]
```

Comment:
I would prefer that we just list out beta and nightly explicitly (I forget if this is a regex or what, too, it might be using shell-like globs in which case this won't work).

Reply:
Yes, we are currently using globs for this.

Reply:
Ah okay, seems like globset does support this notation. I would still prefer two entries in this array.

Reply:
This means I'll have to modify the behaviour of Triagebot. I implemented it in an "AND" fashion, so that all the labels in the array are required. So, if we think we'll never have to combine multiple labels such that all of them are required, I can just open a PR in Triagebot to switch that. Otherwise we'll have to find another solution. What do you think? cc @spastorino

Reply:
Ah okay, makes sense. I think that's probably good to leave in. Mind changing this to use `-{beta, nightly}` presuming that's supported?

Reply:
I've read the documentation of `glob` and I don't think it is supported. I will check again though.

Reply:
I don't think it is supported. See [the `glob` documentation](https://docs.rs/glob/0.3.0/glob/struct.Pattern.html) and [the test](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=9fefc2c8c7c0e91a181600a7e56d325e) I just did in the playground.

Reply:
Ah, okay. Maybe we should switch to globset which seems more featureful (or just go to regex directly).